gpii/node_modules/settingsHandlers/src/settingsHandlerUtilities.js

```diff
@@ -298,6 +298,15 @@ gpii.settingsHandlers.writeFile = function (content, options) {
  */
 gpii.settingsHandlers.handleFileSolutionEntry = function (solutionEntry, modifier, parser, isWrite) {
     var options = solutionEntry.options;
+    var initialContent = options.initialContent;
+    var createFile = options.createFile;
+    //create file
+    if(createFile){
+        var exists = fs.existsSync(options.filename);
+        if(!exists){
+            fs.writeFileSync(options.filename, initialContent, options.encoding || "utf-8");
+        }
+    }
```

Comment:
Could you maybe put a more meaningful comment on there, such as "Create solutions file on disk if it doesn't exist already".

You'll also want to fix up the spacing on the if statements so that it matches the rest of the file.

```
Instead of 'if(createFile){'  you'd want spacing such as 'if (createFile) {'
```